dashboard/app/models/learning_goal.rb

```diff
@@ -35,4 +35,20 @@ def generate_key
     return if key.present?
     self.key = SecureRandom.uuid
   end
+
+  def summarize
```

Comment:
Is this used anywhere? (asking because I think I created a function with the same name 😅 )

Reply:
oh it's used below in `summarize_for_rubric_edit`! Any reason not to combine these functions?

Reply:
I had relied on how activity_section.rb and lesson_activity.rb was set-up which is how this got its current form.  On second look, it seems like `summarize` was used more than once in each of those files, so, yes I agree on combining them.